alacritty/Cargo.toml

```diff
@@ -7,7 +7,7 @@ description = "A fast, cross-platform, OpenGL terminal emulator"
 readme = "README.md"
 homepage = "https://github.com/alacritty/alacritty"
 edition = "2021"
-rust-version = "1.70.0"
+rust-version = "1.72.0"
```

Comment:
A bit ahead of debian, but we likely don't care? 0.8.7 of ahash will work just fine.

Reply:
We also have some time until making another release. I was hoping that until then 1.72.0 is supported.

Reply:
I guess, just add your macOS changes you did in other PR and it should be good.